Index: lib/GeoExt.js
===================================================================
--- lib/GeoExt.js	(revision 2635)
+++ lib/GeoExt.js	(working copy)
@@ -124,27 +124,12 @@
             "GeoExt/Lang.js"
         );
 
-        var agent = navigator.userAgent;
-        var docWrite = (agent.match("MSIE") || agent.match("Safari"));
-        if(docWrite) {
-            var allScriptTags = new Array(jsfiles.length);
-        }
+        var len = jsfiles.length;
+        var allScriptTags = new Array(len);
         var host = getScriptLocation() + "lib/";    
-        for (var i=0, len=jsfiles.length; i<len; i++) {
-            if (docWrite) {
-                allScriptTags[i] = "<script src='" + host + jsfiles[i] +
-                                   "'></script>"; 
-            } else {
-                var s = document.createElement("script");
-                s.src = host + jsfiles[i];
-                var h = document.getElementsByTagName("head").length ? 
-                           document.getElementsByTagName("head")[0] : 
-                           document.body;
-                h.appendChild(s);
-            }
+        for (var i=0; i<len; i++) {
+            allScriptTags[i] = "<script src='" + host + jsfiles[i] +"'></script>"; 
         }
-        if (docWrite) {
-            document.write(allScriptTags.join(""));
-        }
+        document.write(allScriptTags.join(""));
     }
 })();
